pkg/protobuf/app/app.proto

```diff
@@ -119,4 +120,15 @@ message UnsetEnvRequest {
     repeated string env_vars = 2;
 }
 
+message SetAutoScaleRequest {
+    string name = 1;
+	
+	message AutoScale {
+        int32 cpu_target_utilization = 1;
```

Comment:
Why are you using `cpu_target_utilization` on protobuf specification if you don't currently use?